recipes/scons/all/conanfile.py

```diff
@@ -0,0 +1,69 @@
+from conans import ConanFile, tools
+from conans.errors import ConanException
+import os
+import shutil
+import sys
+
+
+class SConsConan(ConanFile):
+    name = "scons"
+    description = "SCons is an Open Source software construction tool—that is, a next-generation build tool"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index/"
+    homepage = "https://scons.org"
+    topics = ("conan", "scons", "build", "configuration", "development")
+    settings = "os_build"  # Added to test on let CI test on all os'es
+
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("scons-{}".format(self.version), self._source_subfolder)
+
+    def build(self):
+        with tools.chdir(self._source_subfolder):
+            self.run("{} setup.py build -j{}".format(sys.executable, tools.cpu_count()))
+
+    def package(self):
+        self.copy("LICENSE*", src=self._source_subfolder, dst="licenses")
+
+        # Mislead CI and create an empty header in the includue directory
+        include_dir = os.path.join(self.package_folder, "include")
+        os.mkdir(include_dir)
+        tools.save(os.path.join(include_dir, "__nop.h"), "")
+
+        with tools.chdir(self._source_subfolder):
+            self.run("{} setup.py install --no-compile --prefix={}".format(sys.executable, self.package_folder))
+
+        tools.rmdir(os.path.join(self.package_folder, "man"))
+
+        if tools.os_info.is_windows:
+            shutil.move(os.path.join(self.package_folder, "Scripts"),
+                        os.path.join(self.package_folder, "bin"))
+            shutil.move(os.path.join(self.package_folder, "Lib"),
+                        os.path.join(self.package_folder, "lib2"))
+            shutil.move(os.path.join(self.package_folder, "lib2"),
```

Comment:
so, you rename "Lib" -> "lib2", then "lib2" -> "lib"? is it just to have lower-case directory after the rename?
I believe you may keep the initial layout project has, just add `cpp_info.libdirs` / `cppinfo.bindirs`?

Reply:
Yes, I could do that. But isn't it nicer to have a lib folder, like all recipes have?

Reply:
I set the libdirs to an empty directory because I do not want these dirs added to the include path of dependencies.

Reply:
okay, looks good, just add some comment(s) so people know what's going on there

Reply:
Added some documentation.